src/app/auth.guard.ts

```diff
@@ -5,13 +5,23 @@ import { OAuthService } from 'angular-oauth2-oidc';
 
 @Injectable()
 export class AuthGuard implements CanActivate {
-  constructor(protected oAuthService: OAuthService, private router: Router, private oauthService: OAuthService) {
-  }
+  constructor(
+    protected oAuthService: OAuthService,
+    private router: Router,
+    private oauthService: OAuthService
+  ) {}
 
   canActivate(
-    next: ActivatedRouteSnapshot,
-    state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
+    _next: ActivatedRouteSnapshot,
+    _state: RouterStateSnapshot
+  ): boolean {
+    // TODO: Uncomment out when login is ready see ticket WF-343
+    // if (this.oauthService.hasValidAccessToken()) {
```

Comment:
I would keep this link in place, no need to comment it out

Reply:
that code does not work. After a while the app gets stuck and It is no longer possible to login. It needs more investigation